framework/Dockerfile

```diff
@@ -34,6 +33,30 @@ ENV IDRINTH_API_BENCH_VERSION=$BUILD_VERSION
 ENV IDRINTH_API_BENCH_BUILT=$BUILD_TIME
 
 RUN mkdir -p /usr/local/lib/node_modules/@idrinth/api-bench
+RUN git clone https://github.com/idrinth-api-bench/framework.git \
+    && npm ci \
+    && clean-modules clean --yes \
+    && mkdir -p /usr/local/lib/node_modules/@idrinth/api-bench \
+    && cp --recursive /api-bench/framework/* /usr/local/lib/node_modules/@idrinth/api-bench/ \
```

Comment:
why is the copy happening before the cleanups?

Reply:
cp is not happening after cleanups?

Reply:
what do you thing 41-56 are?

Reply:
Moved cpy to proper location